db/warehouse/migrate/20240228192937_add_bulk_flag_to_service_type.rb

```diff
@@ -0,0 +1,5 @@
+class AddBulkFlagToServiceType < ActiveRecord::Migration[6.1]
+  def change
+    add_column :CustomServiceTypes, :bulk, :boolean, default: false, null: false, comment: 'whether to support bulk service assignment for this type in the hmis application'
```

Comment:
suggestion: maybe use a more descriptive column name?
```suggestion
    add_column :CustomServiceTypes, :supports_bulk_assignment, :boolean, default: false, null: false, comment: 'whether to support bulk service assignment for this type in the hmis application'
```